### PR TITLE
 Updated libwebp to 0.6.1

### PR DESCRIPTION
**Pull request by radarhere**
A test failed after the update with an average pixel value different of 21.3764, so I increased the threshold to 25.

### CODE REVIEW

File: Tests/test_file_webp_alpha.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def test_write_unsupported_mode_PA(self):
         image.getdata()
         target = Image.open(file_path).convert("RGBA")
 
-        self.assert_image_similar(image, target, 10.0)
+        self.assert_image_similar(image, target, 25.0)
```
Comment (wiredfool) [inline Tests/test_file_webp_alpha.py]: That's a pretty big jump for a .0.1 release, are we sure that we're not missing something there?

Comment (radarhere) [reply Tests/test_file_webp_alpha.py]: https://github.com/webmproject/libwebp/blob/master/NEWS
> - 11/24/2017: version 0.6.1
>   This is a binary compatible release.
>   * lossless performance and compression improvements + a new 'cruncher' mode
>     (-m 6 -q 100)
>   * ARM performance improvements with clang (15-20% w/ndk r15c, issue #339)
>   * webp-js: emscripten/webassembly based javascript decoder
>   * miscellaneous bug & build fixes (issue #329, #332, #343, #353, #360, #361,
>     #363)
>   Tool updates / additions:
>     added webpinfo - prints file format information (issue #330)
>     gif2webp - loop behavior modified to match Chrome M63+ (crbug.com/649264);
>                '-loop_compatibility' can be used for the old behavior

Here's a copy of the generated file from my local machine using 0.6.1.
transparent

**Comment by wiredfool** [comment]
This is now a blocker for the fedora-27 docker tests. https://github.com/python-pillow/docker-images/pull/20 and #2895

File: Tests/test_file_webp_alpha.py (hunk 1 above)
Comment (wiredfool) [reply Tests/test_file_webp_alpha.py]: I have no idea where those issues actually reside on the web, other than not on github, and not in the repo it's cloned from. 

Anyway, it looks like there was a YUV->RGB change in 0.52 that could potentially account for that.

**Pull request merged**